pkg/apis/nsx.vmware.com/v1alpha1/externaladdressbinding_types.go

```diff
@@ -0,0 +1,39 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type ExternalAddressBindingSpec struct {
```

Comment:
How about call it AddressBinding, in case later we will support Project (Private_TGW) IP too?

Reply:
Fixed.